packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -753,12 +764,40 @@ export const RemoteServerFactory = {
                     // purpose so we don't want to overwrite the header for those calls.
                     if (incomingRequest.path?.match(options.applySLASPrivateClientToEndpoints)) {
                         proxyRequest.setHeader('Authorization', `Basic ${encodedSlasCredentials}`)
-                    } else if (incomingRequest.path?.match(/\/oauth2\/trusted-agent\/token/)) {
-                        // /oauth2/trusted-agent/token endpoint auth header comes from Account Manager
-                        // so the SLAS private client is sent via this special header
-                        proxyRequest.setHeader('_sfdc_client_auth', encodedSlasCredentials)
                     }
-                }
+                },
+                onProxyRes: responseInterceptor((responseBuffer, proxyRes, req, res) => {
+                    try {
+                        let data = responseBuffer.toString('utf8')
+                        // If the SLAS response has no body, just return the empty buffer
+                        if (!data) {
+                            return responseBuffer
+                        }
+
+                        // If the response from SLAS has a body, it's expected to be in json format
+                        let body = JSON.parse(data)
+                        const message = body.message
+
+                        // If the message contains the string "user not found", return a 200 OK response
+                        // so it is not obvious that the user does not exist.
+                        if (SLAS_USER_NOT_FOUND_ERROR.test(message)) {
```

Comment:
Another approach you might consider is to respond with a 200 OK for both 200's coming from the passwordless endpoint and 404s (i.e. respond with 200 for both).

This check is a "magic string" that is dependent on SLAS exact messaging not changing and also is testing ALL responses it seems so other endpoints contain this substring may be affected (messages that may not be related to user enumeration)